alma/alma-8.x/common/install_amd_libs.sh

```diff
@@ -10,13 +10,15 @@ AOCL_VERSION=$(jq -r '.aocl.version' <<< $amd_metadata)
 AOCL_SHA256=$(jq -r '.aocl.sha256' <<< $amd_metadata)
 
 TARBALL="aocl-linux-aocc-${AOCL_VERSION}.tar.gz"
-AOCL_DOWNLOAD_URL=https://azhpcstor.blob.core.windows.net/azhpc-images-store/${TARBALL}
+BASE_AOCL_VERSION=${AOCL_VERSION:0:3}
```

Comment:
Is this just grabbing the major.minor version?

Reply:
Yes. In the case of 4.2.0, it grabs 4.2 (this would break down if it were 4.12, but I haven't had any evidence of that. we'll go down that road if we get there)

Reply:
Yeah, that's what I was leaning towards. Leaving this for later, but this should be good for up to a three parted version string
`echo $VERSION | grep -Eo '[0-9]+\.[0-9]+'`